Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector.yml

```diff
@@ -26,8 +26,8 @@ configuration:
   required: false
   type: 8
 - display: The maximum number of events per fetch.
-  additionalinfo: The maximum number of events to fetch every time fetch is being executed. This number must be divisible by 100 due to Saas-Security api limitations. Default is 100.
-  defaultvalue: 100
+  additionalinfo: The maximum number of events to fetch every time fetch is being executed. This number must be divisible by 100 due to Saas-Security api limitations. Default is 1000. In case this is empty, all available events will be fetched.
```

Comment:
please make sure to check the case where we set it to empty that we dont get a default of 0

Reply:
clearly you are right, should be tested